example/zipkin_like/context.py

```diff
@@ -25,13 +25,92 @@
 
 import os
 import opentracing.context
-import opentracing.standard.context
 
 from .span import SAMPLED_FLAG, DEBUG_FLAG
 from .sampler import MAX_ID_BITS
 
+class BaseTraceContext(object):
```

Comment:
(this is definitely the wrong end state, but it's better than having this outside of zipkin_like in `opentracing.standard.context` where I found it)